test/services/company-officers/companyOfficers.service.test.ts

```diff
@@ -54,21 +54,47 @@ describe('CompanyOfficersService', () => {
 
       const secretary: CompanyOfficer = { ...generateCompanyOfficer(), officerRole: 'secretary' }
```

Comment:
is that a valid role? if so, can we introduce enum fo this one?

Reply:
I see this is present in API.

Reply:
It is a role but it isn't valid fo closing a company, so we don't actually use it outside of testing